runtime/lua/vim/snippet.lua

```diff
@@ -397,7 +397,7 @@ local function setup_autocmds(bufnr)
       local cursor_row, cursor_col = cursor_pos()
 
       -- The cursor left the snippet region.
-      local snippet_range = get_extmark_range(bufnr, M._session.extmark_id)
+      local snippet_range = get_extmark_range(bufnr, M._current_session.extmark_id)
```

Comment:
the old name seems fine. and is less verbose :)
```suggestion
      local snippet_range = get_extmark_range(bufnr, M._session.extmark_id)
```

Reply:
Fair enough, I blame C# for making me use extra verbose names lol